src/selectors/history.js

```diff
@@ -42,15 +43,17 @@ export const accountHistorySelector = createSelector(
   },
 );
 
-export const smartAccountHistorySelector = createSelector(
+export const archanovaAccountHistorySelector = createSelector(
   historySelector,
   accountsSelector,
   (history, accounts) => {
-    const userHasSW = userHasSmartWallet(accounts);
-    if (!userHasSW) return [];
-    const smartAccountId = getActiveAccountId(accounts);
-    if (!smartAccountId) return [];
-    return orderBy(history[smartAccountId] || [], ['createdAt'], ['desc']);
+    const userHasArchanovaAccount = findFirstArchanovaAccount(accounts);
+    if (!userHasArchanovaAccount) return [];
+
+    const archanovaAccountId = getActiveAccountId(accounts);
```

Comment:
How do we know that active account is actually archanova?

Reply:
Remember this piece, it had to be little different, it shouldn't check for active account ID, but check for found Archanova account ID, fixed, thanks 👍